src/content/app/species-selector/state/speciesSelectorSlice.ts

```diff
@@ -61,6 +61,7 @@ export type SpeciesSelectorState = {
   };
   ui: {
     isSelectingStrain: boolean;
+    isRemovingSpecies: boolean;
```

Comment:
Woah.

I have a bunch of questions here, which I missed when I was reviewing this PR previously:

- why do we want to keep the species removal state in redux?
- why would we use species selector (which is a different app in our code) for that purpose?

I think the temporary state of species being in the process of removal can be kept as local state within one of the components of the Species page.

Reply:
1. When Remove species button is clicked we need to hide `Find a gene` link. This was as per the initial xd. We do not need this for the new design. So I will remove. Thanks for pointing.
2. We cannot access the local state of one component in another. So I used redux.